customization/dispvm-customization.md

```diff
@@ -65,6 +65,144 @@ In order to do that, select "Qube settings" entry in selected base AppVM, go to
 
 Note that currently only applications whose main process keeps running until you close the application (i.e. do not start a background process instead) will work. One of known examples of incompatible applications is GNOME Terminal (shown on the list as "Terminal"). Choose different terminal emulator (like XTerm) instead.
 
+### Create Custom sys-net sys-firewall and sys-usb DispVMs ###
+
+Users have the option of creating customized DispVMs for the `sys-net`, `sys-firewall` and `sys-usb` VMs. In this configuration, a fresh VM instance is created each time a DispVM is launched. Functionality is near-identical to the default VMs created following a new Qubes’ installation, except the user benefits from a non-persistent filesystem.
+
+Functionality is not limited, users can:
+
+   * Set custom firewall rulesets and run Qubes VPN scripts. 
+   * Set DispVMs to autostart at system boot.
+   * Attach PCI devices with the `--persistent` option. 
+
+Using DispVMs in this manner is ideal for untrusted qubes which require persistent PCI devices, such as USB VMs and NetVMs.
+
+_**Note:**_ Users who want customized VPN or firewall rulesets must create a seperate dvm for use by each DispVM. If dvm customization is not needed, then a single dvm is used as a template for all DispVMs.
+ 
+#### Create and configure the dvm from which the DispVM will be based on ####
+
+   1. Create the dvm 
+
+    [user@dom0 ~]$ qvm-create -P <pool_name> --template <template_name> --class AppVM --label gray <dvm-name>
```

Comment:
I'd leave out `-P <pool_name>` (maybe note it in a comment, if at all), because for most cases it's ok to use default pool. Same for the template.